react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,20 +1,48 @@
-import React from 'react';
+import React, { Component } from 'react';
+
+import getMatches from '../../../services/MatchesService';
 
 import styles from './styles.module.scss';
 import Board from './components/Board';
 
-function Game () {
-  return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
-      </div>
-    </div>
+const Spinner = require('react-spinkit');
+
+class Game extends Component {
+  state = {
+    matches: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    getMatches.getMatches().then(response => {
+      this.setState({ matches: response.data, loading: false });
+    });
+  }
+
+  renderList = item => (
```

Comment:
you can move this outside the component, because this is a pure function and doesn't use `this` anywhere in it.